backend/src/workspaces/dto/create-workspace.dto.ts

```diff
@@ -2,7 +2,12 @@ import { ApiProperty } from "@nestjs/swagger";
 import { MinLength } from "class-validator";
 
 export class CreateWorkspaceDto {
-	@ApiProperty({ description: "Title of project to create", type: String, minLength: 2 })
+	@ApiProperty({
+		description: "Title of project to create",
+		type: String,
+		minLength: 2,
+		maxLength: 120,
+	})
```

Comment:
`@ApiProperty` only defines properties by the OpenAPI spec. You should use the `class-validator` decorator for validation.